internal/proxy/meta_cache.go

```diff
@@ -1087,6 +1097,19 @@ func (m *MetaCache) unsafeInitPolicyInfo(info []string, userRoles []string) {
 		}
 		m.userToRoles[user][role] = struct{}{}
 	}
+	for _, g := range privilegeGroups {
+		if m.privilegeGroups[g.GroupName] == nil {
+			m.privilegeGroups[g.GroupName] = make(map[string]struct{})
+		}
+		for _, priv := range g.Privileges {
+			if !util.IsPrivilegeNameDefined(priv.Name) {
```

Comment:
unnecessary check, the privilegeGroup info comes from rootcoord, rootcoord already validate it's content.

Reply:
so this func wont't return error